test/completion/pep0484_typing.py

```diff
@@ -256,6 +256,20 @@ def testdict(x):
 for value in x.values():
     #? int()
     value
+
+WrappingType = typing.NewType('WrappingType', str) # Chosen arbitrarily
+y = WrappingType(0) # Per https://github.com/davidhalter/jedi/issues/1015#issuecomment-355795929
+#? WrappingType()
+y
+
+def testnewtype(y):
+    """
+    :type y: WrappingType
+    """
+    #? WrappingType()
+    y
+    #? ["upper"]
+    y.u
```

Comment:
Please also add a test where `WrappingType2 = typing.NewType()`. This is a potential issue in your approach.

Reply:
Should the desired behavior for `WrappingType2` be "arbitrary subclass of `object`"? That's what made sense to me, since it has to be a type, and with no arguments, it's not narrowed down.